config/ses.toml

```diff
@@ -24,3 +24,6 @@ target = "devel7"
       storage        = "http://download.suse.de/ibs/SUSE/Products/Storage/7/x86_64/product/"
       storage_devel  = "http://download.suse.de/ibs/Devel:/Storage:/7.0/SLE_15_SP2"
       storage_update = "http://download.suse.de/ibs/SUSE:/SLE-15-SP2:/Update:/Products:/SES7/standard/"
+
+    [ses.devel7.registry]
+      url = "registry.suse.de/suse/sle-15-sp2/update/products/ses7/charts/rook-ceph:1.4.5-1.4"
```

Comment:
This should be something like `rook_ceph_chart` as the config path at the moment implies that it is just a registry url, not a specific chart/package.

Additionally, should we be using `:latest` or something since this is the devel set?